dfget/util/file_util.go

```diff
@@ -35,33 +37,75 @@ func CreateDirectory(dirPath string) error {
 
 // DeleteFile deletes a file not a directory.
 func DeleteFile(filePath string) error {
-	return nil
+	return os.Remove(filePath)
```

Comment:
`os.Remove` will delete a file or a directory, but this function `DeleteFile` just deletes a file not a directory.

Reply:
Besides, the intention should be more clear: what if the provided filepath is a directory, are you going to delete all files under that directory?

Reply:
Thanks for advice, will fix it